cmd/minishift/cmd/start.go

```diff
@@ -87,6 +82,13 @@ var minishiftToClusterUp = map[string]string{
 
 // init configures the command line options of this command
 func init() {
+	startCmd = &cobra.Command{
+		Use:   commandName,
+		Short: "Starts a local OpenShift cluster.",
+		Long:  `Starts a local single-node OpenShift cluster on the specified hypervisor.`,
```

Comment:
Nothing I originally changed, but I agree that "on the specified hypervisor" seems odd. To be honest I am not sure why we single out the hypervisor. `start` has a whole bunch of important parameters. I think I prefer to remove this clause altogether.